gradio/routes.py

```diff
@@ -778,7 +784,6 @@ def read_main():
         # Then run `uvicorn run:app` from the terminal and navigate to http://localhost:8000/gradio.
     """
     blocks.dev_mode = False
-    blocks.root = path[:-1] if path.endswith("/") else path
```

Comment:
This is essentially deprecating the `path` argument of `mount_gradio_app`. I'm on board since it is superfluous. Let's update the docstring and maybe print a warning.

Reply:
Well we still need the `path` argument to specify what route to mount the `gradio` app within the larger FastAPI app (see line 793), so we shouldn't deprecate. But we no longer need to use the `path` parameter to set the `root` here.

Reply:
Makes sense!